sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -102,7 +115,37 @@ public List<String> inputCollectionNames() {
 
   @Override
   public List<String> outputCollectionNames() {
-    return Lists.newArrayList("output");
+    return Arrays.asList("output", "errors");
+  }
+
+  public static class ErrorFn extends DoFn<byte[], Row> {
```

Comment:
nit: I think putting this under the SchemaTransform is a better way since it's closer to where the class is needed.